### DISCUSSION TITLE
Directory Landing Page: Content and Design

### DISCUSSION THREAD

**Discussion by kquirapas** [web3phl/directory#58] [Brainstorm]
Let's finalize the content and the design (wireframe, figma) of the landing page for our directory.

Let the following objectives guide the decision making

# Objectives of Landing Page
1. Give an overview of the core purpose of the directory
    - Serve as the one-stop list of Filipino-Led Web 3 initiatives (projects, company, etc.) that users can use to gather information and confirm validity and reputation of project.
    
2. Guide them into how to sign up to get listed in the directory

3. Highlight the directory as a community-powered initiative and they're free to contribute (provide link to show them how)

**Comment by jnebab** [comment]
I've always liked how SkyMavis implemented their get started page. I guess, we can get some ideas from them for our own signup journey -> https://welcome.skymavis.com/
